Polyhedron/demo/Polyhedron/Plugins/PMP/Join_and_split_polyhedra_plugin.cpp

```diff
@@ -164,6 +171,8 @@ void Polyhedron_demo_join_and_split_polyhedra_plugin::on_actionSplitPolyhedra_tr
         CGAL::copy_face_graph(filter_graph, *new_graph);
         new_polyhedra.push_back(new_graph);
       }
+      //sort polyhedra by number of faces
+      new_polyhedra.sort(Compare());
```

Comment:
Note for next time: in the demo, we can use C++14, with lambdas.
```C++
  new_polyhedra.sort([](FaceGraph* mesh1, FaceGraph* mesh2)
                     {
                        return num_faces(*mesh1) < num_faces(*mesh2);
                     });
```
where the lambda expression is equivalent to `Compare()`:
```C++
[](FaceGraph* mesh1, FaceGraph* mesh2)
{
  return num_faces(*mesh1) < num_faces(*mesh2);
}
```